sillsdev/libpalaso
# Get implicit scripts from SLDR alltags.txt

* parse alltags.txt to get implicit scripts, SLDR-to-Palaso tag mapping, and available LDML files
* only download alltags.txt if it is newer than the embedded alltags.txt
* update alltags.txt to latest version
* update IetfLanguageTag to use implicit scripts from alltags.txt
* add OfflineSldrAttribute to make it easier for unit tests to use SLDR in offline mode
* remove LanguageSubtag.ImplicitScriptCode
* SIL-compatible normalization is no longer necessary
* synchronize access to SLDR cache directory


[<img src="https://reviewable.io/review_button.png" height=40 alt="Review on Reviewable"/>](https://reviewable.io/reviews/sillsdev/libpalaso/318)
